stubs/tomli.pyi

```diff
@@ -0,0 +1,9 @@
+# Author: Felix Fontein <felix@fontein.de>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2022, Ansible Project
+
+from typing import Callable, Any
+
+
+def loads(__s: str, *, parse_float: Callable[[str], Any] = float) -> dict[str, Any]: ...
```

Comment:
Don't these already exist upstream?

Reply:
Yes, but tomli isn't installed, and both pyre and mypy complain.

Reply:
Can we add it to to `dev-dependencies`?

Reply:
Why? It's not really needed.

Reply:
The `types-*` packages are for packages that are actual dependencies (and that are needed to do something). tomli is not needed to run anything during development.

Reply:
FWIW, we pull in `types-toml`

Reply:
I guess.